elements/rh-footer/test/rh-footer.spec.ts

```diff
@@ -125,6 +142,7 @@ const GLOBAL_FOOTER = html`
       <a href="#">*We’ve updated our privacy statement effective December 30, 202X.</a>
     </div>
   </rh-global-footer>
+  <link rel="stylesheet" href="/elements/rh-footer/rh-footer-lightdom.css" />
```

Comment:
```suggestion
  <link rel="stylesheet" href="/elements/rh-footer/rh-footer-lightdom.css">
```

i recently learned that html5 doesn't need to /> self-closing tags